src/renderer/main/panels/configuration/Configuration.store.js

```diff
@@ -458,7 +457,7 @@ function create_configuration_manager() {
   }
 
   user_input.subscribe((ui) => {
-    store.update((store) => {
+    update((store) => {
```

Comment:
This update overwrites the standard store operation as the current code suggests. Such overwrite wrapping is totally doable, but I think it's prone to user errors, specially when revisiting code. The direct `store.update(...)` and `update(...)` is easy to miss.

One possible indicator could be that we say `alteredUpdate(...)` instead of simply `update(...)`. Same improved naming scheme could applied to `set` function as well.

In this case the `setIndentation()` is required after `set` and `update`, so maybe we should make `setAndIndentate` and `updateAndIndendate`.

My point is, looking at this code, I can totally understand why it was easy to miss the specific set/update command required.

Reply:
Fixed it in the latest 2 commit. Ambiguous naming and function usage was resolved by using better function naming.